src/components/NotesMindMapView/NotesMindMapViewSagas.js

```diff
@@ -83,7 +85,7 @@ function renderChildren(children, targetNote, visNetwork) {
   });
 }
 
-function createEmptyChild({ data: { parentId, visNetwork } }) {
+function createEmptyChild({ data: { parentId, visNetwork } }, editNote = () => {}) {
```

Comment:
This is not relevant to the change note parent issue, and it's creating even more mess.

It will be fixed after the refactoring is finished: https://github.com/DarthVanger/NeuralNotes/pull/52/files

Please exclude this change from your PR.